MAINTAINERS.md

```diff
@@ -126,23 +221,19 @@ in this branch. The release tags will be human-meaningful, the release branch ne
   scripts/have-news HEAD^ > have-news.md
   ```
 
-- [ ] Create the release PR.
-
-  The above should have pushed state to GitHub to let you create a
-  new PR to merge the release branch back to the main branch. The PR
-  name should follow the convention of previous releases, which is
-  currently `chore(release): publish _release_label_`.  Paste
-  `have-news.md` as the description of the PR.  Follow the example
-  of previous releases for any other details of the PR.
+- [ ] Create the release PR:
+  - Source branch: the [timestamped release branch](#user-content-release-branch)
+  - Target branch: the [_**base branch**_](#assign-release-parameters)
+  - PR name: "chore(release): publish [_**release short label**_](#assign-release-parameters)"
+  - PR description: the contents of `have-news.md` from the previous step
 
-  (Note that the `have-news.md` file might be too large for a Git commit
-  message if you've gone too long without making a release.)
+  Follow the example of previous releases for any other details of the PR.
 
-  Creating this PR will also kick off the CI tests.
+  Creating this PR will initiate the CI tests.
```

Comment:
We should probably have the PR labelled `force:integration` (at least until we have better CI logic to detect these kind of release PRs).

The idea is that integration tests should pass before we publish, which happens before we attempt to merge.